From 426fa7f6c0ad7db9c6b18ef9b2247be224e0ea01 Mon Sep 17 00:00:00 2001
From: Vishal Verma <vishal.l.verma@intel.com>
Date: Wed, 15 Jun 2022 16:48:10 -0600
Subject: [PATCH 209/217] ndctl: remove obsolete m4 directory

With the conversion to meson, the m4 directory, which may have held
symlinks to libtool.m4 and friends, is no longer needed. Remove it.

Link: https://lore.kernel.org/r/20220615224813.523053-3-vishal.l.verma@intel.com
Cc: Dan Williams <dan.j.williams@intel.com>
Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
---
 m4/.gitignore | 6 ------
 1 file changed, 6 deletions(-)
 delete mode 100644 m4/.gitignore

diff --git a/m4/.gitignore b/m4/.gitignore
deleted file mode 100644
index 8bab51c..0000000
--- a/m4/.gitignore
+++ /dev/null
@@ -1,6 +0,0 @@
-libtool.m4
-ltoptions.m4
-ltsugar.m4
-ltversion.m4
-lt~obsolete.m4
-
-- 
2.27.0

